docs/conf.py

```diff
@@ -19,7 +19,7 @@
 copyright = "Block Research Group - ETH Zurich"
 author = "Tom Van Mele"
 
-release = "release = '0.17.2'"
+release = "0.17.2"
```

Comment:
Late to the review party, but this change needs a matching change on the `https://github.com/compas-dev/compas/blob/master/.bumpversion.cfg` file otherwise it will not update it.